pkg/testing/integration_register.go

```diff
@@ -85,7 +85,7 @@ var _ = Suite(&CockroachDB{
 	},
 })
 
-// time-log app for csi volumesnapshot
+// TimeLogCSI registers time-log app for csi volumesnapshot.
```

Comment:
I think these comments should be improved, these are just types they don't register the app right?

Reply:
done